File: season_1/programming_lab/TDE_01/exercise_01.c
#include <stdio.h>

int main() {
    int numero;

    printf("Bem-vindo ao teste de numeros!!\n\n");
    printf("Digite um numero de 1 a 7: ");

    if (scanf("%d", &numero) != 1) {
        printf("Erro: Voce nao digitou um algarismo\n");
        return 1;
    }

    switch (numero) {
        case 1:
            printf("Domingo\n");
            break;
        case 2:
            printf("Segunda-feira\n");
            break;
        case 3:
            printf("Terca-feira\n");
            break;
        case 4:
            printf("Quarta-feira\n");
            break;
        case 5:
            printf("Quinta-feira\n");
            break;
        case 6:
            printf("Sexta-feira\n");
            break;
        case 7:
            printf("Sabado\n");
            break;
        default:
            printf("Dia invalido\n");
    }

    return 0;
}

File: season_1/programming_lab/TDE_01/exercise_01.py
print('Bem-vindo ao teste de números!!\n')

try:
    numero = int(input('Digite um número de 1 a 7:\n'))

    match numero:
        case 1:
            print('Domingo')
        case 2:
            print('Segunda-feira')
        case 3:
            print('Terça-feira')
        case 4:
            print('Quarta-feira')
        case 5:
            print('Quinta-feira')
        case 6:
            print('Sexta-feira')
        case 7:
            print('Sábado')
        case _:
            print('Dia inválido')
except ValueError:
    print('Erro: Você não digitou um algarismo')
